app/src/main/java/com/example/frolic/Event.java

```diff
@@ -1,43 +1,69 @@
 package com.example.frolic;
+import android.graphics.Bitmap;
 import android.util.Log;
 
-import com.google.firebase.firestore.auth.User;
-
 import java.util.Date;
 
 import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
 
 public class Event {
-    private Organizer organizer;
+    private String eventId;
+    private String organizerId;
     private String eventName;
     private String eventDesc;
-    private ArrayList<Entrant> confirmed;
+    private ArrayList<Entrant> confirmed = new ArrayList<>();
     private Facility facility;
-    private int maxEntrants;
+    private int waitlistLimit;
     private int maxConfirmed;
     private Date eventDate;
     private Date enrollDate;
     private final LotterySystem lottery;
+    private double price;
```

Comment:
I'm not entirely sure if we should have a "price" value -- I understand that a scenario includes someone wanting to set up an event with a price to enter, but there isn't a user story for this as far as I can see.
Maybe if we did add this functionality, we could do something similar to Start.gg, where events can be signed up for and if there's a price, the website simply says "I understand that I will pay at the venue and that start.gg isn't responsible for transactions" because I strongly doubt hazel wants us to figure out a way to actually send payments to anyone through the app